autopts/ptsprojects/stack/layers/bap.py

```diff
@@ -107,6 +108,19 @@ def wait_bis_synced_ev(self, broadcast_id, bis_id, timeout, remove=True):
             lambda ev: (broadcast_id, bis_id) == (ev['broadcast_id'], ev['bis_id']),
             timeout, remove)
 
+    def wait_bis_stopped_ev(self, broadcast_id, bis_id, reason, timeout, remove=True):
```

Comment:
Since `reason` is seemingly optional to set, suggest to make it
```suggestion
    def wait_bis_stopped_ev(self, broadcast_id, bis_id, timeout, reason=None, remove=True):
```

Reply:
I was thinking about it, but I want @sjanc's input on this since that breaks with the argument-ordering of many of the other functions in this file.